cinema/permissions.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework import permissions
+
+
+class IsAdminOrIfAuthenticatedReadOnly(permissions.BasePermission):
+    def has_permission(self, request, view):
+        return bool(
+            request.method in
+            permissions.SAFE_METHODS and request.user.is_authenticated
```

Comment:
`getattr(request.user, 'is_authenticated', False)`
I'm not sure it would work correctly if we won't be authenticated, it always should work as expected, and we don't probably expect AttributeError.